### DISCUSSION TITLE
Squelch much more sensitive

### DISCUSSION THREAD

**Discussion by LHunti** [egzumer/uv-k5-firmware-custom#127] [Q&A] [answered]
Hi.

Is it possible to set the squelch level for an even weaker reception, or does this already have hardware limitations? Because often in monitor mode you can still hear weaker broadcasts relatively well.

Thanks for the answer.

**Comment by egzumer** [comment]
If I set squelch to 1 the noise is getting through, how much more sensitive can it be?

**Reply by LHunti** [reply]
Yes indeed, the noise does come through, which is already a problem with normal use, but when I try to catch a distant broadcast, I sometimes miss a reception. My experience is that a Baofeng UV-5R is perhaps more sensitive, able to produce appreciable reception even at -135 dBm, although it is true that it has an AT1846S chip, which also works in a narrower frequency range. Actually, I was interested in lowering the squelch level, because I often search for distant broadcasts, but I don't want to constantly listen to noise. I hope you understand what I want, because unfortunately my English is not very good.

**Reply by egzumer** [reply]
If you lower the squelch it will just let more noise through. There is no point in lowering it even more.

**Reply by LHunti** [reply]
I understand, thanks for the answer and also for the custom firmware you made. I've tried many, but in the end this is my favorite and I'm not alone :) Have a nice day.

**Comment by f5gkw** [comment, marked as answer]
Maybe you can think about to add a preamplifier ( LNA) .
JPh aka F5GKW

**Reply by LHunti** [reply]
hmm, that's a good idea, thanks, I'll experiment with it.
